### PR TITLE
py-spaCy and py-catalogue: Update and Submission

### PR DESCRIPTION
**Pull request by essandess**
py-spaCy and py-catalogue: Update and Submission

* Update py-spaCy
* Submit new dependency py-catalogue

###### Type(s)

- [x] update
- [x] submission
- [ ] bugfix
- [ ] enhancement
- [ ] security fix

###### Tested on

macOS 10.x
Xcode 8.x

###### Verification 
Have you

- [x] followed our [Commit Message Guidelines](https://trac.macports.org/wiki/CommitMessages)?
- [x] squashed and [minimized your commits](https://guide.macports.org/#project.github)?
- [x] checked that there aren't other open [pull requests](https://github.com/macports/macports-ports/pulls) for the same change?
- [ ] referenced existing tickets on [Trac](https://trac.macports.org/wiki/Tickets) with full URL?

- [x] checked your Portfile with `port lint`?
- [ ] tried existing tests with `sudo port test`?
- [x] tried a full install with `sudo port -vst install`?
- [x] tested basic functionality of all binary files?

### CODE REVIEW

**Comment by cjones051073** [comment]
thanks, been meaning to find time to look into this one...

File: python/py-catalogue/Portfile [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+                    size    8310
+
+python.versions     35 36 37 38
+
+if {${name} ne ${subport}} {
+    depends_build-append \
+                    port:py${python.version}-setuptools
+
+    depends_run-append \
+                    port:py${python.version}-pytest
```
Comment (reneeotten) [inline python/py-catalogue/Portfile]: @essandess I'm pretty sure that ```pytest``` is not a runtime dependency for this port... or? Additionally, you're missing a dependency on ```importlib_metadata``` for anything below PY38 ([see uptream's setup.cfg](https://github.com/explosion/catalogue/blob/6f44510c6c3e10216b99076f631f93cf2132be2f/setup.cfg#L34)), packaged in MacPorts under ```pyXY-importlib-metadata```. Please be more careful about dependencies as I've  noted in some of your other PRs as well.

Comment (essandess) [reply python/py-catalogue/Portfile]: I'll edit this dependency in https://github.com/macports/macports-ports/pull/5862

**Comment by museum-future** [comment]
this port still doesn't build
https://ports.macports.org/port/py37-spaCy/builds

**Pull request merged**